<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActorDialog</class>
 <widget class="QDialog" name="ActorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>direction:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>action:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_direction">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>down</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>up</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cb_action">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tbl_stats">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>life</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>manna</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>stamina</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>wealth</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>sprite sheet</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>position:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lbl_name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_position">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lbl_position</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>primary:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_primary_fight">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ActorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ActorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_direction</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ActorDialog</receiver>
   <slot>directionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_action</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ActorDialog</receiver>
   <slot>actionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>directionChanged()</slot>
  <slot>actionChanged()</slot>
 </slots>
</ui>
